### PR TITLE
Core Team minutes 14/5/19

### PR DESCRIPTION
**Pull request by BKiddo**

### CODE REVIEW

File: 2018/2019-05-14.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Agenda
+1. Finance
+2. Core Team disbandment
+3. Kit
+
+## Minutes
+1. Tyler is still waiting on several receipts, including StageSoc, pizza, and Kickstart van hire. Andy BS will chase up volunteer bank details if not provided, and forward Diane's email detailing scaffolding payment to Tyler. The total spend for the competition cycle should be under 10k.
+2. The Trustees have suggested a date of 6th June for disbandment. We need to consider having interim people to answer email queries etc.
+3. There are still a mix of full and partial kits that have not been returned past the dealine. Jake will chase this up. It would be useful to have another kit day, and there appear to be missing power boards, along with the box of Smallpeice things.
```
Comment (trickeydan) [inline 2018/2019-05-14.md]: > and there appear to be missing power boards, along with the box of Smallpeice things.

Can anyone give more context to this? I'm guessing that SR's power boards are missing, and SRO's smallpeice box?

Comment (RealOrangeOne) [reply 2018/2019-05-14.md]: Yes it's SRs power boards.

The smallpeice items related to @Tyler-Ward 's comment on that apparently there were some things which SRO couldn't find, and wondered if they had been accidentally shipped to Newbury (which they didm't)

Comment (trickeydan) [reply 2018/2019-05-14.md]: >  if they had been accidentally shipped to Newbury (which they didm't)

We need that confirmed, as we are in fact missing a RUB full of new, uninventorised smallpeice equipment.

Comment (RealOrangeOne) [reply 2018/2019-05-14.md]: >  which they didn't

See previous message for confirmation they weren't shipped to Newbury.

We checked the labels on each RUB as it was put into the container so the labels were facing outwards, we would have noticed if one wasn't SRs (which we didn't).

Comment (trickeydan) [reply 2018/2019-05-14.md]: > > which they didn't
> 
> See previous message for confirmation they weren't shipped to Newbury.
> 
> We checked the labels on each RUB as it was put into the container so the labels were facing outwards, we would have noticed if one wasn't SRs (which we didn't).

I trust that you believe this, however human error is definitely a thing. If we can double check next time someone is in the container that would be great. I'll open a task for this, not urgent as long as it occurs before SP2019

**Pull request merged**